new_game.py

```diff
@@ -0,0 +1,139 @@
+    def practice(self, answer):
+        print 'Please type the answer 3x, each on its own line.\n'
+        count = 0
+        while count < 3:
+            word = raw_input('> ')
+            if word == answer:
+                count += 1
+            else:
+                print 'Make sure your spelling is correct.'
+        game_on = False
+
+
+class Main(object):
+
+    def __init__(self):
+        self.wins = 0
+        self.attempts = 0
+
+    def clear_screen(self):
+        if os.name == "posix":
+            # Unix/Linux/MacOS/BSD/etc
+            os.system('clear')
+        elif os.name in ("nt", "dos", "ce"):
+            # DOS/Windows
+            os.system('CLS')
+        else:
+            # Fallback for other operating systems.
+            print '\n' * numlines
+    # Prompt to choose easy = number version or hard = word version
+    def start(self):
+        self.clear_screen()
+        game_on = True
+        while game_on:
+            choice = raw_input('\nWould you like the easy or hard version? \n').lower()
+            if choice.startswith('h'):
+                self.reset_score()
+                self.word_game()
+            elif choice.startswith('e'):
+                self.reset_score()
+                self.num_game()
+                print 'Given the definition, choose the correct number that',
+                'corresponds to the word. You have two tries.'
+            else:
+                print 'I don\'t know what that means.'
+    # Choose the number of questions
+    def num_questions(self):
+        while True:
+            try:
+                num = int(raw_input('How many questions would you like? (1-20) '))
+            except:
+                print "That is not a valid number."
+                continue
+            else:
+                break
+        return num
+    # Word game layout
+    def word_game(self):
+        num = self.num_questions()
+        print '\nGiven the definition, type the correct word. You have one try.'
+        raw_input('Press Enter to start...')
+        for i in range(num):
+            time.sleep(2)
+            self.clear_screen()
+            answer = data.answer()
+            data.definition(answer)
+            data.choices(answer)
+
+            turns = 0
+            while turns < 1:
+                guess = raw_input('\nAnswer: ')
+                if guess.isdigit():
+                    print "Please type the correct word as written."
+                elif guess == answer:
+                    print 'Correct!'
+                    self.wins += 1
+                    break
+                else:
+                    print 'Incorrect.'
+                    turns += 1
+                    if turns == 1:
+                        print 'The answer was %s.' % (answer)
+                        data.practice(answer)
+            self.attempts += 1
+        print 'Your score is %d/%d.' % (self.wins, self.attempts)
+        print '-' * 10
+        game_on = False
```

Comment:
this should be self.game_on

To access variables within your class you need to say self.blah